bouncer/src/repo/core/model/bson/repo_bson_factory.cpp

```diff
@@ -893,37 +893,34 @@ TextureNode RepoBSONFactory::makeTextureNode(
 	const uint32_t    &byteCount,
 	const uint32_t    &width,
 	const uint32_t    &height,
-	const int         &apiLevel)
+	const int         &apiLevel,
```

Comment:
Can we swap the order? parentIDs go before apiLevel, every other function in this file that accepts a parentID array take it this way.

Reply:
this broke code on the assimp loader , but i refactored it to match

Reply:
Fair enough, I'm just going to tweak that quickly, the apiLevel stuff are kind of useless legacy that we don't really use. even if we do it should just be using the default, so I've just altered to function call so it doesn't pass it in